src/base.rs

```diff
@@ -455,7 +455,7 @@ pub struct MacAddress {
 /// order (i.e., big endian). Note that no special alignment restrictions are
 /// defined by the standard specification.
 #[repr(C)]
-#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+#[derive(Clone, Copy, Debug, Eq, PartialEq, Default)]
 pub struct Ipv4Address {
     pub addr: [u8; 4],
 }
```

Comment:
`std::net::Ipv4Addr` does not implement `Default`. Do you happen to know why? I think `INADDR_ANY` / all-zero is a suitable default, so I am fine with this. I just wondered why `std` does not have this.

Reply:
I just noticed that now as well. I would have expected the default to be `UNSPECIFIED`.